core/src/cubos/core/data/package.cpp

```diff
@@ -494,14 +535,20 @@ impl::Unpackager::Unpackager(const Package& pkg) : pkg(pkg)
     // Do nothing.
 }
 
+#define UNEXPECTED(expected, found)                                                                                    \
+    do                                                                                                                 \
```

Comment:
Why do a `do {...} while(false);`? Wouldn't it work with just `{...}`?

Reply:
It's common practice to use `do {...} while(false)` to ensure that the a `;` must follow the macro, as if it was a normal statement.

Reply:
I see, the more you know :slightly_smiling_face: